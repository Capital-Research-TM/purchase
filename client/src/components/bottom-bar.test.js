import React from 'react';
import { configure, shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import BottomBar from './bottom-bar.jsx';

// configure({ adapter: new Adapter() });

const open = [true, false];
const up = [true, false];

const testRender = function(openStatus, upStatus) {
  it('bottom bar renders correctly for state: ' + openStatus + ' and ' + upStatus, () => {
    const temp = renderer
      .create(<BottomBar open={openStatus} up={upStatus} balance={59} />)
      .toJSON();
    expect(temp).toMatchSnapshot();
  });
};

for (var openStatus in open) {
  for (var upStatus in up) {
    testRender(openStatus, upStatus);
  }
};
