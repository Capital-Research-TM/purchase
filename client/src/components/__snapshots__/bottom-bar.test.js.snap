// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bottom bar renders correctly for state: 0and 0 1`] = `
<div
  className="sc-bwzfXH fTajkR"
  open="0"
>
  <div
    className="sc-bxivhb bELKrm"
  >
    <div
      className="sc-bdVaJa hETcTg"
      open="0"
    >
      $
      59.00
      Buying Power Available
    </div>
  </div>
</div>
`;

exports[`bottom bar renders correctly for state: 0and 1 1`] = `
<div
  className="sc-bwzfXH fTajkR"
  open="0"
>
  <div
    className="sc-bxivhb bELKrm"
  >
    <div
      className="sc-bdVaJa hETcTg"
      open="0"
    >
      $
      59.00
      Buying Power Available
    </div>
  </div>
</div>
`;

exports[`bottom bar renders correctly for state: 1and 0 1`] = `
<div
  className="sc-bwzfXH fTajkR"
  open="1"
>
  <div
    className="sc-bxivhb bELKrm"
  >
    <div
      className="sc-bdVaJa hETcTg"
      open="1"
    >
      $
      59.00
      Buying Power Available
    </div>
  </div>
</div>
`;

exports[`bottom bar renders correctly for state: 1and 1 1`] = `
<div
  className="sc-bwzfXH fTajkR"
  open="1"
>
  <div
    className="sc-bxivhb bELKrm"
  >
    <div
      className="sc-bdVaJa hETcTg"
      open="1"
    >
      $
      59.00
      Buying Power Available
    </div>
  </div>
</div>
`;

exports[`bottom bar renders correctly for state: true and 0 1`] = `
<div
  className="sc-bwzfXH fTajkR"
  open="0"
>
  <div
    className="sc-bxivhb bELKrm"
  >
    <div
      className="sc-bdVaJa hETcTg"
      open="0"
    >
      $
      59.00
      Buying Power Available
    </div>
  </div>
</div>
`;

exports[`bottom bar renders correctly for state: true and 0 2`] = `
<div
  className="sc-bwzfXH fTajkR"
  open="1"
>
  <div
    className="sc-bxivhb bELKrm"
  >
    <div
      className="sc-bdVaJa hETcTg"
      open="1"
    >
      $
      59.00
      Buying Power Available
    </div>
  </div>
</div>
`;

exports[`bottom bar renders correctly for state: true and 1 1`] = `
<div
  className="sc-bwzfXH fTajkR"
  open="0"
>
  <div
    className="sc-bxivhb bELKrm"
  >
    <div
      className="sc-bdVaJa hETcTg"
      open="0"
    >
      $
      59.00
      Buying Power Available
    </div>
  </div>
</div>
`;

exports[`bottom bar renders correctly for state: true and 1 2`] = `
<div
  className="sc-bwzfXH fTajkR"
  open="1"
>
  <div
    className="sc-bxivhb bELKrm"
  >
    <div
      className="sc-bdVaJa hETcTg"
      open="1"
    >
      $
      59.00
      Buying Power Available
    </div>
  </div>
</div>
`;
